drivers/can/can_net.c

```diff
@@ -183,6 +186,33 @@ static inline int attach_mcast_filter(struct net_can_context *ctx)
 	return filter_id;
 }
 
+#ifdef CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR
```

Comment:
Minor nitpick but we are preferring `#if defined(xxx)` in networking code, Could use change all of the `#ifdef` to those?